### PR TITLE
add kafka connect for data ingestion post

### PR DESCRIPTION
**Pull request by jaehyeon-kim**
### Description
PR for community blog series.

Ingest data from Kafka into OpenSearch using Kafka Connect
* Part 1 - Local Development
* Part 2 - AWS Deployment

It just includes Part 1 for now for initial discussion. Part 2 will be include shortly after. 

### Issues Resolved
* [Kafka Connect for OpenSearch Integration](https://github.com/opensearch-project/project-website/issues/1882)

### Check List
- [x] Commits are signed per the DCO using --signoff

By submitting this pull request, I confirm that my contribution is made under the terms of the BSD-3-Clause License.

### CODE REVIEW

**Comment by dtaivpp** [comment]
Hello! Can you please do the following to resolve DCO? Thank you!

1. In your local branch, run: `git rebase HEAD~2 --signoff`
2. Force push your changes to overwrite the branch: `git push --force-with-lease origin main`

**Comment by jaehyeon-kim** [comment]
Part 2 is added.

**Comment by jaehyeon-kim** [comment]
Hi, any update on this PR?

**Comment by pajuric** [comment]
Hi @jaehyeon-kim - we need to break this up into two PRs to avoid pushing both Part 1 and Part 2 live at the same time. In the meantime, I have asked PM to review the content.

File: _posts/2023-09-28-Ingest-data-from-Kafka-into-OpenSearch-using-Kafka-Connect-Part-1-Local-Development.markdown [hunk 1]
```diff
@@ -0,0 +1,534 @@
...
+
+When it comes to ingesting data from Apache Kafka into OpenSearch, OpenSearch has a tool called [Data Prepper](https://opensearch.org/docs/latest/data-prepper/index/) and Amazon OpenSearch Service has a feature called [Amazon OpenSearch Ingestion](https://docs.aws.amazon.com/opensearch-service/latest/developerguide/ingestion.html). Alternatively we can use [Kafka Connect](https://kafka.apache.org/documentation/#connect), which is a tool for scalably and reliably streaming data between Apache Kafka and other systems. On AWS, we can run fully managed Kafka workload using [Amazon MSK Connect](https://aws.amazon.com/msk/features/msk-connect/).
```
Comment (kkondaka) [inline _posts/2023-09-28-Ingest-data-from-Kafka-into-OpenSearch-using-Kafka-Connect-Part-1-Local-Development.markdown:20]: You should probably mention Managed version of Data-Prepper called "OpenSearch Ingestion Service" (https://docs.aws.amazon.com/opensearch-service/latest/developerguide/ingestion.html) in this context?

**Review by dlvenable — CHANGES_REQUESTED** [review]

**Comment by pajuric** [comment]
@jaehyeon-kim - checking in on your blog and seeing if you were able to address the comments above?

**Pull request closed without merging**